src/features/perfBrowser/usePerfBrowserColors.tsx

```diff
@@ -11,13 +11,13 @@ type ColorDiscriminantProps = {
 export const PERF_BROWSER_COLORSCHEMES = [
     'primary',
     'orange',
-    'blue',
-    'pink',
     'green',
+    'pink',
     'yellow',
     'cyan',
     'red',
     'purple',
+    'blue',
```

Comment:
Do we still keep the blue ? I know it's last in the list so unless we get 9 differents lines it won't be a problem but just in case

Reply:
this is not really blocking, just wanted to point this out

Reply:
It's still different and better to have more colors i think 👍